src/EnergyPlus/HeatBalanceManager.cc

```diff
@@ -1217,17 +1219,17 @@ namespace HeatBalanceManager {
                 {
                     std::string const &SELECT_CASE_var = AlphaName(1);
```

Comment:
This is what `getEnumValue` is for.

Reply:
yeah, let me change this to use getEnumValue